Source/WebCore/loader/ImageLoader.cpp

```diff
@@ -212,6 +213,7 @@ void ImageLoader::updateFromElement(RelevantMutation relevantMutation)
         resourceRequest.setInspectorInitiatorNodeIdentifier(InspectorInstrumentation::identifierForNode(m_element));
 
         auto request = createPotentialAccessControlRequest(WTFMove(resourceRequest), WTFMove(options), document, crossOriginAttribute);
+        request.setFetchPriorityHint(options.fetchPriorityHint);
```

Comment:
Probably not needed since we are passing options when creating request.

Reply:
I think so as well, removed.